Content.Server/GameObjects/Components/Nutrition/SmokingComponent.cs

```diff
@@ -62,30 +60,22 @@ private SharedBurningStates CurrentState
                 switch (_currentState)
                 {
                     case SharedBurningStates.Lit:
-                        _clothingComponent.EquippedPrefix = "lit";
+                        _clothingComponent!.EquippedPrefix = "lit";
                         _clothingComponent.ClothingEquippedPrefix = "lit";
                         break;
                     default:
-                        _clothingComponent.EquippedPrefix = "unlit";
+                        _clothingComponent!.EquippedPrefix = "unlit";
                         _clothingComponent.ClothingEquippedPrefix = "unlit";
                         break;
```

Comment:
using `!` here is a big nono! please just add a null check before it instead of ignoring nullability, this WILL throw if no clothingcomponent is present